ezgui/src/backend_glow.rs

```diff
@@ -158,10 +100,6 @@ impl<'a> GfxCtxInnards<'a> {
             }
         }
     }
-
-    pub fn finish(self) {
-        self.windowed_context.swap_buffers().unwrap();
```

Comment:
Most of the remaining divergence in code between glow-wasm and glow-native are around window/windowed_context, so this couldn't be in this file with the "common" glow code.